src/index.js

```diff
@@ -47,10 +47,14 @@ const toJSON = (obj) => JSON.parse(JSON.stringify(obj))
 export default function (schema, opts) {
   const options = merge({}, defaultOptions, opts)
 
+  // get _id type from schema
+  options._idType = schema.tree._id.type
```

Comment:
Yes, mongoose will add the _id property to the schema if it's not already defined.